src/App.js

```diff
@@ -1,39 +1,63 @@
 //TODO: STEP 1 - Import the useState hook.
-import React from "react";
+import React, { useState } from "react";
 import "./App.css";
 import BottomRow from "./BottomRow";
 
 function App() {
   //TODO: STEP 2 - Establish your applictaion's state with some useState hooks.  You'll need one for the home score and another for the away score.
 
+  const [tigers, tigersValue] = useState(0);
+  const [lions, lionsValue] = useState(0);
+
   return (
     <div className="container">
       <section className="scoreboard">
         <div className="topRow">
           <div className="home">
             <h2 className="home__name">Lions</h2>
+            <div className="home_score">{lions}</div>
 
             {/* TODO STEP 3 - We need to change the hardcoded values in these divs to accept dynamic values from our state. */}
 
-            <div className="home__score">32</div>
+            <div className="home__score">{lions}</div>
           </div>
           <div className="timer">00:03</div>
           <div className="away">
             <h2 className="away__name">Tigers</h2>
-            <div className="away__score">32</div>
+            <div className="away__score">{tigers}</div>
           </div>
         </div>
         <BottomRow />
       </section>
       <section className="buttons">
         <div className="homeButtons">
           {/* TODO STEP 4 - Now we need to attach our state setter functions to click listeners. */}
-          <button className="homeButtons__touchdown">Home Touchdown</button>
-          <button className="homeButtons__fieldGoal">Home Field Goal</button>
+          <button
+            className="homeButtons__touchdown"
+            onClick={() => lionsValue(lions + 3)}
+          >
+            Home Touchdown
+          </button>
+          <button
+            className="homeButtons__fieldGoal"
+            onClick={() => lionsValue(lions + 7)}
+          >
+            Home Field Goal
+          </button>
         </div>
         <div className="awayButtons">
-          <button className="awayButtons__touchdown">Away Touchdown</button>
-          <button className="awayButtons__fieldGoal">Away Field Goal</button>
+          <button
+            className="awayButtons__touchdown"
+            onClick={() => tigersValue(lions + 3)}
+          >
+            Away Touchdown
+          </button>
+          <button
+            className="awayButtons__fieldGoal"
+            onClick={() => tigersValue(lions + 7)}
+          >
+            Away Field Goal
+          </button>
         </div>
       </section>
     </div>
```

Comment:
perfect! Wish i had more feedback for you but this assignment was just too easy for an amazing dev like you!